src/pages/en/guides/rss.md

```diff
@@ -6,11 +6,139 @@ description: An intro to RSS in Astro
 
 Astro supports fast, automatic RSS feed generation for blogs and other content websites. For more information about RSS feeds in general, see [aboutfeeds.com](https://aboutfeeds.com/).
 
-You can create an RSS feed from any Astro page that uses a `getStaticPaths()` function for routing. Only dynamic routes can use `getStaticPaths()` today (see [Routing](/en/core-concepts/routing)).
+## Using `@astrojs/rss` (recommended)
 
-> We hope to make this feature available to all other pages before v1.0. As a workaround, you can convert a static route to a dynamic route that only generates a single page. See [Routing](/en/core-concepts/routing) for more information about dynamic routes.
+The `@astrojs/rss` package provides helpers for generating RSS feeds using [API endpoints](/en/core-concepts/astro-pages/#non-html-pages). This unlocks both static builds _and_ on-demand generation when using an [SSR adapter](/en/guides/server-side-rendering/#enabling-ssr-in-your-project).
 
-Create an RSS Feed by calling the `rss()` function that is passed as an argument to `getStaticPaths()`. This will create an `rss.xml` file in your final build based on the data that you provide using the `items` array.
+First, install `@astrojs/rss` using your preferred package manager:
+
+```bash
+# npm
+npm i @astrojs/rss
+# yarn
+yarn add @astrojs/rss
+# pnpm
+pnpm i @astrojs/rss
+```
+
+To generate your first RSS feed, create an `rss.xml.js` file under your `src/pages/` directory. `rss.xml` will be the output URL, so feel free to rename this if you prefer.
+
+Now, import the `rss` helper from the `@astrojs/rss` package. We will explore each of the parameters below:
+
+```js
+// src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+
+export const get = () => rss({
+    // `<title>` field in output xml
+    title: 'Buzz’s Blog',
+    // `<description>` field in output xml
+    description: 'A humble Astronaut’s guide to the stars',
+    // list of `<item>`s in output xml
+    // simple example: generate items for every md file in /src/pages
+    // see "Generating items" section for required frontmatter and advanced use cases
+    items: import.meta.glob('./**/*.md'),
+    // (optional) inject custom xml
+    customData: `<language>en-us</language>`,
+  });
+```
+
+### Generating `items`
+
+The `items` field accepts either:
+1. [A list of RSS feed objects](#1-list-of-rss-feed-objects), each with a `link`, `title`, `pubDate`, and optional `description` and `customData` fields.
+2. [An `import.meta.glob(...)` result](#2-importmetaglob-result). **Only use this for `.md` files in the `src/pages/` directory!**
```

Comment:
```suggestion
2. [An `import.meta.glob(...)` result](#2-importmetaglob-result). **Only use this for `.md` files within the `src/pages/` directory!**
```

Tagging @delucis and @aFuzzyBear on this very tiny nit. We have tended to use "within the `src/pages/` directory when describing Markdown that lives in any level of subfolders. Our valient (?) attempt to distinguish from "directly in." Might be totally unnecessary, but it's a thing that we've done in general, so just pointing it out.

Reply:
aye this looks good to me buddy 🙂

Reply:
Okay, will keep that consistency in mind!